docs/docs/administration/08-advanced/02-roles-and-permissions.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Roles and permissions
+description: Guide how to properly configure roles and permissions
+---
+
+## Introduction
+
+Understanding roles and permissions in PILOS is essential for maintaining effective access control and system security.
+
+---
+
+## Permissions
+
+Permissions are the smallest unit of access control in PILOS. They define what actions a user can perform in the system, except in rooms, where a user's role also influences their abilities.
```

Comment:
actions allowed to perform are not abilities? not 100% sure what this is supposed to mean